### ISSUE TITLE
Allow "keep_on_remove"

### ISSUE THREAD

**Issue by Rodri9o** [Terraform-VMWare-Modules/terraform-vsphere-vm#65]
Dear @Arman-Keyoumarsi ,

Is "keep_on_remove" for VM disks supported by this module ?
https://registry.terraform.io/providers/hashicorp/vsphere/latest/docs/resources/virtual_machine#keep_on_remove

I checked the variables for something related to this settings but could not identify anything.

Once again, thanks for the amazing module.
Cheers!
Rod

**Comment by Arman-Keyoumarsi** [comment]
No, It is not defined yet. We are always looking for contributors. I am happy to approve your PR if you are willing to implement it, otherwise I will keep it in the list of todos for the next release :).

**Comment by dmpopoff** [comment]
> I am happy to approve your PR if you are willing to implement it :).
your welcome :)
[data disks](https://github.com/Terraform-VMWare-Modules/terraform-vsphere-vm/pull/161)
[template disks](https://github.com/Terraform-VMWare-Modules/terraform-vsphere-vm/pull/162)
